packages/taiko-client/driver/chain_syncer/blob/syncer_test.go

```diff
@@ -111,6 +111,10 @@ func (s *BlobSyncerTestSuite) TestInsertNewHead() {
 }
 
 func (s *BlobSyncerTestSuite) TestTreasuryIncomeAllAnchors() {
+	// TODO: Temporarily skip this test case when use l2_reth node.
+	if os.Getenv("L2_NODE") == "l2_reth" {
```

Comment:
Let's use `L2_RETH_NODE`  to make it more clear?

Reply:
`L2_NODE` is a common name and the value may be l2_reth or l2_geth.